velox/common/encode/Base64.cpp

```diff
@@ -546,4 +546,104 @@ void Base64::decodeUrl(
   decodedOutput.resize(decodedSize.value());
 }
 
+// static
+// The implementation of this function is inspired by the Java Base64 mime
+// decoder:
+// https://github.com/openjdk/jdk/blob/master/src/java.base/share/classes/java/util/Base64.java#L810
```

Comment:
Not sure whether we had the similar issue as https://github.com/facebookincubator/velox/pull/10930#discussion_r1801224802. cc: @assignUser